torch/csrc/jit/codegen/cuda/kernel.cpp

```diff
@@ -19,6 +19,8 @@ namespace {
 //!
 class KernelIrScanner : private OptOutDispatch {
  public:
+  size_t expr_cnt = 0;
```

Comment:
what is this field? A more descriptive name + comment may help (no need for aggressive abbreviations, we're not using 70s host machines). How about 'all_expressions_count` ?